apps/src/templates/export/expo/app.json.ejs

```diff
@@ -2,15 +2,15 @@
   "expo": {
     "name": "<%- appName %>",
     "description": "<%- appName %> from Code.org Code Studio.",
-    "slug": "<%- appName.replace(/([^a-zA-Z0-9_\-]+)/gi, '-') %>",
+    "slug": "project-<%- projectId.toLowerCase().replace(/([^a-zA-Z0-9_\-]+)/gi, '-') %>",
```

Comment:
```suggestion
    "slug": "project-<%- projectId.toLowerCase().replace(/[^\w-]+/g, '-') %>",
```
Simplified regex:
- The capture group `()` around your whole expression is unneeded.
- `\w` "Word characters" is equivalent to `[a-zA-Z0-9_]`
- They hyphen doesn't need to be escaped anymore.
- `i` case-insensitive flag isn't needed because it's already encompassed by the word-character class and because you've already converted the string to lowercase.

One step further, but with a slight behavior change, is to allow the hyphen to be replaced and just use the `\W` non-word character class:
```js
.replace(/\W+/g, '-')
```
The difference in behavior in this case is that consecutive hyphens will collapse, where with your original implementation they will not:
```
// Before
'A-b-$-c--d' => 'a-b---c--d'
// After
'A-b-$-c--d' => 'a-b-c-d'
```
Not sure which you prefer.